src/reducers/shelf/filter.test.ts

```diff
@@ -1,175 +1,147 @@
 
-import {Schema} from 'compassql/build/src/schema';
 import {OneOfFilter, RangeFilter} from 'vega-lite/build/src/filter';
 import {TimeUnit} from 'vega-lite/build/src/timeunit';
 import {
   FILTER_ADD, FILTER_CLEAR, FILTER_MODIFY_EXTENT, FILTER_MODIFY_MAX_BOUND,
   FILTER_MODIFY_MIN_BOUND, FILTER_MODIFY_ONE_OF, FILTER_MODIFY_TIME_UNIT, FILTER_REMOVE
 } from '../../actions';
-import {DEFAULT_SHELF_UNIT_SPEC} from '../../models/shelf';
-import {ShelfUnitSpec} from '../../models/shelf/spec';
 import {filterReducer} from './filter';
 
-const rangeFilter: RangeFilter = {field: 'q1', range: [1437978615, 1501137015]};
+const range = [1437978615, 1501137015];
```

Comment:
It seems like most of these constants are used only once.   If that's the case, it's better to just put them inline where there test are.  Putting them up here separate them from the code, making it hard to reader and understand the tests.

(We should only put reused stuff up here.)